builder/azure/arm/builder.go

```diff
@@ -14,6 +14,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2021-11-01/compute"
```

Comment:
Will there be any needed changes besides the import when switching to the HashiCorp Azure SDK?
 - If so it might help to port this change over once merged.